backend/src/middlewares/app_cache.ts

```diff
@@ -1,9 +1,12 @@
 import NodeCache from "node-cache";
 import { NextFunction, Request, Response } from "express";
+import { GameSessions } from "../gameSessions";
 
 const cache = new NodeCache();
+const gameSessions = new GameSessions();
 
 export function setAppCache(req: Request, _res: Response, next: NextFunction): void {
   req.appCache = cache;
+  req.gameSessions = gameSessions;
```

Comment:
If we do it like this, isn't the gameSessions object re-initialised at each request?

Reply:
I checked, it keeps the same object between 2 requests 👍